test/functional/wallet_send.py

```diff
@@ -142,7 +142,12 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
             return
 
         if locktime:
+            assert_equal(from_wallet.decoderawtransaction(from_wallet.gettransaction(res["txid"])["hex"])["locktime"], locktime)
             return res
+        else:
+            if add_to_wallet:
+                decoded_tx = from_wallet.decoderawtransaction(from_wallet.gettransaction(res["txid"])["hex"])
+                assert_greater_than(decoded_tx["locktime"], 100)
```

Comment:
In 5101f054ce435ac653a3f5e780fca066e2f25088 "test: test sendall and send do anti-fee-sniping"

Shouldn't this check for `getblockcount() - 100`?

Reply:
Done